community/portworx/templates/portworx-controller.yaml

```diff
@@ -112,6 +115,7 @@ spec:
         scheduler.alpha.kubernetes.io/critical-pod: ""
       labels:
         name: portworx-pvc-controller
+        app: portworx-pvc-controller
```

Comment:
We should just use the `name:<value>` labels instead of having both `name` and `app`

Reply:
Done.